source/simulator/checkpoint_restart.cc

```diff
@@ -268,6 +268,8 @@ namespace aspect
   void Simulator<dim>::create_snapshot()
   {
     TimerOutput::Scope timer (computing_timer, "Create snapshot");
+    walltime_from_resume += walltime.wall_time();
```

Comment:
can you add a comment here, why this is necessary?

Reply:
yep